src/modules/Dropdown/Dropdown.js

```diff
@@ -39,7 +124,7 @@ export default class Dropdown extends Component {
 
   render() {
     const options = _.map(this.props.options, (opt, i) => {
-      return <option key={i} value={opt.value}>{opt.text}</option>;
+      return <option key={opt.value} value={opt.value}>{opt.text}</option>;
```

Comment:
Minor fix while I'm in here, index as key is a risky anti-pattern, and since the `value` is a string value we can just use that as the key.